appinventor/blocklyeditor/src/backpack.js

```diff
@@ -440,6 +440,17 @@ Blockly.Backpack.prototype.openBackpackDoc = function(e) {
                                          });
   };
   options.push(backpackDoc);
+
+  // Clear backpack.
+  var backpackClear = {enabled: true};
+  backpackClear.text = Blockly.Msg.BACKPACK_EMPTY;
+  backpackClear.callback = function() {
+    if (Blockly.getMainWorkspace().hasBackpack()) {
+      Blockly.getMainWorkspace().getBackpack().clear();
+    }
+  };
+
+  options.push(backpackClear);
```

Comment:
I recommend that you move the options.push line up one and have an extra line here before the call to Blockly.ContextMenu.show to spatially group it with the creation of the option rather than the opening of the menu.